x/epochs/keeper/abci_test.go

```diff
@@ -17,8 +17,6 @@ import (
 
 // This test is responsible for testing how epochs increment based off
 // of their initial conditions, and subsequent block height / times.
-//
-// TODO: Make a new test for init genesis logic
```

Comment:
InitGenesis basically just shells out to AddEpochInfo. Error on duplicate is tested in its TestEpochsInitGenesis already!